robotoff/cli/main.py

```diff
@@ -998,5 +998,22 @@ def create_migration(
         router.create(name, auto=auto)
 
 
+@app.command()
+def launch_batch_job(
+    job_type: str = typer.Argument(..., help="Type of job to launch. Ex: 'ingredients_spellcheck'"),
+) -> None:
+    """Launch a batch job."""
+    from robotoff.batch import launch_batch_job
+    from robotoff.utils import get_logger
+    from robotoff.types import BatchJobType
+
+    if job_type not in BatchJobType.__members__:
```

Comment:
Nope since job_type is a string and not an enum

```python
from enum import Enum
class Model(Enum):
    a = 1
    b = 2
try: 
    "a" in Model
except Exception as e:
    print(e)

print("a" in Model.__members__) 
```

```
unsupported operand type(s) for 'in': 'str' and 'EnumType'
True
<ipython-input-2-32a1e4cd847a>:6: DeprecationWarning: in 3.12 __contains__ will no longer raise TypeError, but will return True or
False depending on whether the value is a member or the value of a member
  "a" in Model
```

Reply:
Ah yes, usually I do a BatchJobType[job_type] and catch KeyError, but this approach is fine too :+1: